packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -8,66 +8,66 @@ environment:
   flutter: '>=3.13.0 <4.0.0'
 
 dependencies: {{#enable_dev_menu}}
-  alice: ^1.0.0-dev.10
-  alice_dio: ^1.0.4{{/enable_dev_menu}}
+  alice: ^1.0.0-dev.12
+  alice_dio: ^1.0.6{{/enable_dev_menu}}
   collection: ^1.18.0
   copy_with_extension: ^5.0.4{{#enable_pin_code}}
-  crypto: ^3.0.3{{/enable_pin_code}}
-  cupertino_icons: ^1.0.6
-  dio: ^5.3.3
+  crypto: ^3.0.6{{/enable_pin_code}}
+  cupertino_icons: ^1.0.8
+  dio: ^5.7.0
   equatable: ^2.0.5{{#analytics}}
-  firebase_analytics: ^11.3.0{{/analytics}}{{#uses_firebase}}
-  firebase_core: ^3.4.0{{/uses_firebase}}{{#analytics}}
-  firebase_crashlytics: ^4.1.0{{/analytics}}{{#push_notifications}}
-  firebase_messaging: ^15.1.0{{/push_notifications}}
+  firebase_analytics: ^11.3.3{{/analytics}}{{#uses_firebase}}
```

Comment:
Please communicate in the changelog that the project dependency versions are updated

Reply:
Done